### PR TITLE
 Fix for issue #265 - Added simple install script to be used with curl or wget and bash

### PR DESCRIPTION
**Pull request by marcus-oscarsson**
Hey

Simple install script to be used with curl or wget and bash, currently only for Debian packages, that means Debian, and Ubuntu like systems. The system packages also needs to be fairly recent. Can easily be adapted to rpm based systems.

Usage:

```
curl -o- https://raw.githubusercontent.com/mxcube/mxcube3/master/debian_install.sh | bash
wget -qO- https://raw.githubusercontent.com/mxcube/mxcube3/master/debian_install.sh | bash

```

Related to issue #265 

Cheers,
Marcus

### CODE REVIEW

**Comment by meguiraun** [comment]
could you update https://github.com/mxcube/mxcube3/wiki/How-to:-Installation so both wiki and script are consistent in terms of packages?

File: debian-install.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  pip2.7 install -r requirements.txt
+}
+
+install_node() {
+  # nvm
+  command curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh | bash
+  export NVM_DIR="$HOME/.nvm"
+  source "$NVM_DIR/nvm.sh"
+  source "$NVM_DIR/bash_completion"
+  nvm install --lts
```
Comment (meguiraun) [inline debian-install.sh]: why nvm?

Comment (marcus-oscarsson) [inline debian-install.sh]: I found Node Version Manager (nvm) the easiest way to install node, there are maybe others but this is fairly platform independent and makes the script easier to port.

**Comment by marcus-oscarsson** [comment]
No problem ill update the wiki page when the script exists, id like to do some additional test from the repository before i add the instructions to the install page.

**Comment by meguiraun** [comment]
creating an issue so we do not forget to update the wiki

**Pull request merged**